src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoUrgency.json

```diff
@@ -1,23 +1,25 @@
 {
-  "case_number" : "1",
-  "courthouse_id" :  $COURTHOUSE_ID,
-  "courthouse" : "some-courthouse",
-  "case_reporting_restrictions" : [ ],
-  "status" : "Approved",
-  "received" : "2023-06-20T10:00:00Z",
-  "requestor" : {
-    "user_id" : $USER_ACCOUNT_ID,
-    "user_full_name" : "integrationtest.user@example.comFullName"
+  "case_number": "1",
+  "courthouse_id": $COURTHOUSE_ID,
+  "courthouse": "some-courthouse",
```

Comment:
Should this be uppercase?

Reply:
no, this is showing the display_name, so wont be upper-case, as per https://tools.hmcts.net/jira/browse/DMP-3651